noresm2.1/noresm2.1_release_notes.rst

```diff
@@ -3,7 +3,7 @@
 NorESM2.1 release notes
 =======================
 
-The NorESM2.1 release is a technical release that includes a few bug fixes along with numerous code improvements from the NorESM2.0 release (NorESM tag release-noresm2.0.6).
+The NorESM2.1 release (NorESM tag **release-noresm2.1.1**) is a technical release that includes a few bug fixes along with numerous code improvements from the NorESM2.0 release (NorESM tag **release-noresm2.0.6**).
 
```

Comment:
Why do we say noresm2.0.6 here and 2.0.7 in the release text?

Reply:
Good point. I've change it so that we only us noresm2.0.7 in the noresm2.1 doc.